clang/lib/Parse/ParseInit.cpp

```diff
@@ -437,7 +437,9 @@ ExprResult Parser::createEmbedExpr() {
   SourceLocation StartLoc = ConsumeAnnotationToken();
   if (Data->BinaryData.size() == 1) {
     Res = IntegerLiteral::Create(Context,
-                                 llvm::APInt(CHAR_BIT, Data->BinaryData.back()),
+                                 llvm::APInt(CHAR_BIT, Data->BinaryData.back(),
```

Comment:
`llvm::APInt(CHAR_BIT, (uint8_t)Data->BinaryData.back())`, maybe?

Reply:
Done, though I used `(unsigned char)` to match `CHAR_BIT` more closely.